packages/conventional-commits-parser/test/parser.spec.js

```diff
@@ -525,11 +525,12 @@ describe('parser', function() {
     });
 
     it('should parse important notes that start with asterisks (for squash commits)', function() {
-      var text = 'Previously multiple template bindings on one element\n' +
-          '(ex. `<div *ngIf=\'..\' *ngFor=\'...\'>`) were allowed but most of the time\n' +
-          'were leading to undesired result. It is possible that a small number\n' +
-          'of applications will see template parse errors that shuld be fixed by\n' +
-          'nesting elements or using `<template>` tags explicitly.\n' +
+      var expectedText = 'Previously multiple template bindings on one element\n' +
+        '(ex. `<div *ngIf=\'..\' *ngFor=\'...\'>`) were allowed but most of the time\n' +
+        'were leading to undesired result. It is possible that a small number\n' +
+        'of applications will see template parse errors that shuld be fixed by\n' +
+        'nesting elements or using `<template>` tags explicitly.';
```

Comment:
The asterisk-extracted notes text omits the reference text now. This matches the behaviour of other notes-related tests.